pkg/surveyext/editor_test.go

```diff
@@ -14,59 +14,137 @@ import (
 	"github.com/AlecAivazis/survey/v2/terminal"
 	pseudotty "github.com/creack/pty"
 	"github.com/stretchr/testify/assert"
-	"github.com/stretchr/testify/require"
 )
 
-func Test_GhEditor_Prompt(t *testing.T) {
+func testLookPath(s string) ([]string, []string, error) {
+	return []string{os.Args[0], "-test.run=TestHelperProcess", "--", s}, []string{"GH_WANT_HELPER_PROCESS=1"}, nil
+}
+
+func TestHelperProcess(t *testing.T) {
+	if os.Getenv("GH_WANT_HELPER_PROCESS") != "1" {
```

Comment:
What is the purpose of having the `GH_WANT_HELPER_PROCESS` check? Seems like it is a precautionary measure, but I can't think of the case it is guarding against.

Reply:
The `TestHelperProcess` function needs to be skipped during normal test run. It should only be active when invoked as a subprocess